package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@daostack/subgraph",
-  "version": "0.0.35-0",
+  "version": "0.0.36-0",
```

Comment:
why not make it 0.0.35-1 ?

Reply:
Because I try to keep up with the migration repo which is on v36 already

Reply:
not sure it is a good must method . and also I do not see that the migration version is incremented on this pr